catalog-rest-service/src/main/resources/json/schema/entity/data/pipeline.json

```diff
@@ -5,6 +5,32 @@
   "description": "This schema defines the Pipeline entity. A pipeline enables the flow of data from source to destination through a series of processing steps. ETL is a type of pipeline where the series of steps Extract, Transform and Load the data.",
   "type": "object",
   "definitions": {
+    "statusType": {
+      "javaType": "org.openmetadata.catalog.type.StatusType",
+      "description": "Enum defining the possible Status.",
+      "type": "string",
+      "enum": [
+        "successful",
```

Comment:
Should this be CamelCase? Successful, Failed, Pending ?

Also do we need "javaEnum" for this ?